docs/clients/cloud-watch-logs.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: client
+category: clients
+name: CloudWatchLogs
+package: async-aws/cloud-watch-logs
+---
+
+## Usage
+
+### List log streams
+
+```php
+use AsyncAws\CloudWatchLogs\CloudWatchLogsClient;
+use AsyncAws\CloudWatchLogs\Input\DescribeLogStreamsRequest;
+use AsyncAws\CloudWatchLogs\ValueObject\LogStream;
+
+$cloudWatchLogs = new CloudWatchLogsClient();
+
+$result = $cloudWatchLogs->describeLogStreams(new DescribeLogStreamsRequest([
+    'logGroupName' => 'log-group-name',
+]));
+
+/** @var LogStream $stream */
+foreach ($result->getLogStreams() as $stream) {
+    echo '-'.$stream->getLogStreamName().': '.$stream->getUploadSequenceToken().PHP_EOL;
+}
+```
+
+### Push logs
+
+```php
+use AsyncAws\CloudWatchLogs\CloudWatchLogsClient;
+use AsyncAws\CloudWatchLogs\Input\PutLogEventsRequest;
+use AsyncAws\CloudWatchLogs\ValueObject\InputLogEvent;
+
+$cloudWatchLogs = new CloudWatchLogsClient();
+
+$result = $cloudWatchLogs->putLogEvents(new PutLogEventsRequest([
+    'logGroupName' => 'log-group-name',
+    'logStreamName' => 'log-stream-name',
+    'logEvents' => [
+        new InputLogEvent([
+            'timestamp' => date('U.u') * 1000,
+            'message' => 'test message',
+        ]),
+    ],
+    'sequenceToken' => 'sequence-token',
```

Comment:
I don't get the purpose of this property. 

Is it compulsory? if yes can we update the example to help understanding it?
```php
'sequenceToken' => $currentSequenceToken
/* ... */
$currentSequenceToken = $result->getNextSequenceToken();
```

Reply:
Without sequenceToken the API throws with :
```
HTTP 400 returned for "https://logs.eu-west-1.amazonaws.com/".

Code: InvalidSequenceTokenException
Message: The given sequenceToken is invalid. The next expected sequenceToken is: ***
Type: InvalidSequenceTokenException
```

Reply:
I have tested the case when the stream is new (no events yet) it accepts null as sequenceToken

I have updated the example accordingly